CMakeLists.txt

```diff
@@ -276,3 +276,28 @@ if (DOXYGEN_FOUND)
                        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
                        COMMENT "Generating API documentation with Doxygen" VERBATIM)
 endif (DOXYGEN_FOUND)
+
+##############
+# Add a target to the build which will install the pstore components.
+
+add_pstore_install_targets (
+    install-pstore
+    DEPENDS
+        pstore-broker-poker
+        pstore-broker-status
+        #pstore-broker-service
```

Comment:
Should this be conditionally added if the target is windows?

Reply:
You are correct. New patch will include target only if (WIN32 OR CYGWIN OR MINGW).

Reply:
Note that neither CYGWIN nor MINGW are supported by pstore ATM. That's a job for someone...

Reply:
I will use the generic Windows platform specified by PLATFORM_ID

Reply:
Elsewhere we just use "if(WIN32)". Adding support for CYGWIN/MINGW is a separate piece of work.

Reply:
Changing to use WIN32.